janusgraph-core/src/main/java/org/janusgraph/graphdb/configuration/GraphDatabaseConfiguration.java

```diff
@@ -277,8 +278,15 @@
             "Schema related configuration options");
 
     public static final ConfigOption<String> AUTO_TYPE = new ConfigOption<>(SCHEMA_NS,"default",
-            "Configures the DefaultSchemaMaker to be used by this graph. If set to 'none', automatic schema creation is disabled. " +
-                    "Defaults to a blueprints compatible schema maker with MULTI edge labels and SINGLE property keys",
+            "Configures the DefaultSchemaMaker to be used by this graph."
+            + " Either one of the following shorthands can be used: <br>"
+            + " - `default` (a blueprints compatible schema maker with MULTI edge labels and SINGLE property keys),<br>"
+            + " - `none` (automatic schema creation is disabled)<br>"
+            + " - `tp3`<br>"
```

Comment:
_tp3_ wasn't mentioned before here. If it's now mentioned, then there should also be a short explanation of what this is.
To be honest, I didn't know about the existence of this schema maker before and after looking into it, I still don't know its purpose. So, I would just remove its mention here and therefore also _tp3-logging_ below, but if you know about a valid use case for it, then you can of course also just mention that here.

Reply:
I set in documentation in order to be exhaustive, but I do not use tp3 or tp3-logging. This schema marker may be usefull if you want to have by default a LIST cardinality (see https://github.com/JanusGraph/janusgraph/pull/1915#issuecomment-572575405 )
Note: As Tp3DefaultSchemaMaker does not extends JanusGraphDefaultSchemaMaker, using tp3 has other impacts.

I'am fine to remove reference in doc to tp3 and tp3-logging.

Reply:
I have prepared branches with the 2 options: https://github.com/mmadoo/janusgraph/tree/improvement%23191-addTp3doc and https://github.com/mmadoo/janusgraph/tree/improvement%231917-noTp3 which one do you prefer ?

Reply:
I would use the `noTp3` branch because I think that we shouldn't offer too many schema makers if they don't add a clear benefit.